src/amo/components/AddonMoreInfo/index.js

```diff
@@ -19,41 +20,61 @@ export class AddonMoreInfoBase extends React.Component {
   render() {
     const { addon, i18n } = this.props;
 
+    if (!addon) {
+      return (
```

Comment:
The header gets repeated here and the loading styles also don't seem to match up too well. What about using a helper like this?

```js
return this.renderCard({
  homepage: <LoadingText width={55} />,
  supportUrl: <LoadingText width={55} />
  version: <LoadingText width={55} />,
  lastUpdated: <LoadingText width={25} />,
  // etc...
});
```

When you have an add-on, you could just fill it in with real values or nulls as necessary:

```js
return this.renderCard({
  homepage,
  supportUrl,
  version: addon.current_version.version,
  lastUpdated: i18n.sprintf(...),
  // etc...
});
```

Reply:
Yeah, I could make it more DRY.

Also yeah I'll tweak the styles a bit, I just sort of noticed that. Thanks!

Reply:
Oh to be clear though I want the `<dt>` *and* `<dd>`s to appear as `<LoadingText />` because which sections appear are pretty varied, so I don't want to render:

```
<dt>Privacy Policy</dt>
<dd><LoadingText /></dd>
```
because it might turn out the add-on doesn't even have a Privacy Policy. So I'm just rendering a few dt/dd sets.

Reply:
ok, in that case you can at least share the header like:

```js
return this.renderCard(
  <dl className="AddonMoreInfo-contents">
    <dt><LoadingText width={55} /></dt>
    <dd><LoadingText width={25} /></dd>
    <dt><LoadingText width={55} /></dt>
    <dd><LoadingText width={25} /></dd>
  </dl>
);
```

However, I'd think for the purpose of making the loading screen match the final content you'd want to preserve the headers like `<dt>Privacy Policy</dt>`.

Reply:
Indeed, that's what I ended up with.

Unfortunately we can't do that because the headers that appear are quite variable. Nearly everything is conditionally rendered. I should be able to match the size at least with the right margins.